src/shared/bus-util.h

```diff
@@ -49,11 +49,17 @@ struct bus_properties_map {
         size_t offset;
 };
 
+enum {
+        BUS_MAP_COPY_STRING       = 1U << 0, /* If set, strings are copied. Thus, userdata needs to be freed. */
+        BUS_MAP_BOOLEAN_AS_UINT32 = 1U << 1, /* If set, "b" message is stored as uint32_t. If not set, "b" is stored as bool. */
```

Comment:
hmm, this isn't used yet?

Reply:
Yeah, my previous PR already converted all Boolean to bool.
Should we change this to BUS_MAP_BOOLEAN_AS_BOOL or something?

Reply:
Ah, right. Yes, I think that would be good. i.e. BUS_MAP_BOOLEAN_AS_BOOL.

I mean, it's of course a valid question to prefer consistency within sd-bus-related code (like we would if the mapping code would default to map bools as "int"), or to prefer shortness of code (like we do in current git master, as the flag does not need to be specified). However, I think in the long run, being the least confusing is best, and that would mean to opt for bools as ints by default, and bools as bools if the flag is specified, even if that means we have to specify the flag always